merkle-tree/indexed/tests/tests.rs

```diff
@@ -467,3 +468,80 @@ where
 pub fn test_insert_invalid_low_element_poseidon() {
     insert_invalid_low_element::<Poseidon>()
 }
+
+#[test]
+pub fn functional_non_inclusion_test() {
+    let mut relayer_indexing_array =
+        IndexedArray::<Poseidon, usize, INDEXING_ARRAY_ELEMENTS>::default();
+
+    // appends the first element
+    let mut relayer_merkle_tree = reference::IndexedMerkleTree::<Poseidon, usize>::new(
+        MERKLE_TREE_HEIGHT,
+        MERKLE_TREE_CANOPY,
+    )
+    .unwrap();
+    let nullifier1 = 30_u32.to_biguint().unwrap();
+    relayer_merkle_tree
+        .append(&nullifier1, &mut relayer_indexing_array)
+        .unwrap();
+    // indexed array:
+    // element: 0
+    // value: 0
+    // next_value: 30
+    // index: 0
+    // element: 1
+    // value: 30
+    // next_value: 0
+    // index: 1
+    // merkle tree:
```

Comment:
number missing?

Reply:
this is just a header like indexed array above